<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Status</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Motor 1 : NC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Motor 2 : NC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Power offset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="power_offset_spinner">
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.680000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>current multiplication factor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>6.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="MatplotlibWidget" name="mplwidget"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="3">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Motor 2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDial" name="motor_2_dial">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="notchTarget">
                <double>5.000000000000000</double>
               </property>
               <property name="notchesVisible">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDial" name="motor_1_dial">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="wrapping">
                <bool>false</bool>
               </property>
               <property name="notchTarget">
                <double>5.000000000000000</double>
               </property>
               <property name="notchesVisible">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="motor_1_spinner">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QSpinBox" name="motor_2_spinner">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QProgressBar" name="motor_2_power_bar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="dial_label">
               <property name="text">
                <string>Duty Cycle</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Motor 1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QProgressBar" name="motor_1_power_bar">
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
               <property name="textVisible">
                <bool>true</bool>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QCheckBox" name="motor_link">
               <property name="text">
                <string>Link</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Duty Cycle</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Position</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Velocity</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>Duty Cycle</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Position</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Velocity</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="motor_1_fwd_rev">
               <property name="text">
                <string>FWD/REV</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QCheckBox" name="motor_2_fwd_rev">
               <property name="text">
                <string>FWD/REV</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>75</height>
       </size>
      </property>
      <property name="title">
       <string>Commands</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="start_button">
         <property name="toolTip">
          <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start Changing Data to Both of the Motors&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="stop_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="reset_button">
         <property name="text">
          <string>Reset defaults</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>motor_1_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>motor_1_spinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motor_2_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>motor_2_spinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motor_1_spinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>motor_1_dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motor_2_spinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>motor_2_dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>799</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
